docs/source/changes.txt

```diff
@@ -3,6 +3,11 @@
 Changes
 ..............................................................................
 
+0.9: 2018-09-18
```

Comment:
@yeesian would you be willing to change the version to "Next" with no date? I think the release still needs to be decided.